tests/python/test_element_wise.py

```diff
@@ -10,7 +10,7 @@ def _c_mod(a, b):
 
 @pytest.mark.parametrize('lhs_is_mat,rhs_is_mat', [(True, True), (True, False),
                                                    (False, True)])
-@ti.all_archs
+@ti.all_archs_with(fast_math=False)
```

Comment:
Does `pytest.approx` with larger `rel=` and `abs=` solve your issue?

Reply:
Nope, it failed at `//`, which translates to `floor(y / z)`. For `y[1][1]` and `z[1][1]`, they are both `3`, so I guess that the division becomes something like `0.99999`, which gets floored to `0.0` instead of `1.0`. I just tested by setting `y` at a slightly higher value, e.g. `3.0001`, and it worked..